php/README.md

```diff
@@ -0,0 +1,155 @@
+|  `ingress.enabled` | Enables Ingress | `FALSE` |
+|  `ingress.annotation` | Ingress annotations |  |
+|  `ingress.host` | Ingress accepted hostname | nginx.host |
+|  `ingress.port` | Ingress port | service.port or nginx.port |
+|  `ingress.path` | Ingress path |  |
+|  `ingress.preferPaths` | Paths that takes precedence over the ingress.path |  |
+|  `ingress.tls.secretName` | TLS Secret (certificates) name  |  |
+|  `podDisruptionBudget.enabled` | If true, create a pod disruption budget for keeper pods | `FALSE` |
+|  `podDisruptionBudget.minAvailabled` | Minimum number / percentage of pods that should remain scheduled |  |
+|  `podDisruptionBudget.maxAvailabled` | Minimum number / percentage of pods that should remain scheduled |  |
+|  `extras.templates` | Additional raw Kubernetes resources | `{}` |
+|  `test.enabled` | Enables helm test | `TRUE` |
+|  `test.extraContainers` | Additional containers |  |
+
+### Init Containers
+
+Initialize using busybox.
+By default, we use [busybox](https://hub.docker.com/_/busybox), but you can use your container.
+We recommend that you embed the source code in your container and copy it to the container of PHP and NGINX in the initialization process.
+
+|  Parameter | Description | Default |
+| --- | --- | --- |
+|  `busybox.image.repository` | The image repository to pull from | `busybox` |
+|  `busybox.image.tag` | The image tag to pull | `latest` |
+|  `busybox,image.pullPolicy` | Image pull policy | `IfNotPresent` |
+|  `busybox.command` | Initialize command | `["sh", "-c", "echo '<?php phpinfo();' > /var/www/html/index.php"]` |
+|  `busybox.sharedPath` | Path of directory to mount | `sharePath` |
+|  `busybox.extraEnv` | Additional environment variables | `{}` |
+|  `busybox.extraEnvFrom` | Additional envFrom | `{}` |
+|  `busybox.extraVolumeMounts` | Additional volumeMounts | `{}` |
+|  `busybox.secrets` | Additional Secret as a string to be passed to the tpl function | `{}` |
+|  `busybox.templates` | Additional ConfigMap as a string to be passed to the tpl function. | `{}` |
+|  `busybox.annotations` | Grant annotations to ConfigMap of `busybox.templates`, Secrets of `busybox.secrets` | `{}` |
+
+## Nginx Containers
+
+|  Parameter | Description | Default |
+| --- | --- | --- |
+|  `nginx.image.repository` | The image repository to pull from | `nginx` |
+|  `nginx.image.tag` | The image tag to pull | `1.15.1-alpine` |
+|  `nginx.image.pullPolicy` | Image pull policy | `IfNotPresent` |
+|  `nginx.port` | Default public port of NGINX | `80` |
+|  `nginx.livenessProbe` | Overrides the default liveness probe | HTTP `/status` port `7777` |
+|  `nginx.readinessProbe` | Overrides the default readness probe | HTTP `/ping` port `7777` |
+|  `nginx.resources` | Overrides the default resource | `{}` |
+|  `nginx.extraEnv` | Additional environment variables | `{}` |
+|  `nginx.extraEnvFrom` | Additional envFrom | `{}` |
```

Comment:
Values is ```[]``` ?
and
- ```nginx.extraPorts``` as same.
- ```nginx.extraVolumes``` is seemed to be not here
- ```nginx.etraVolumeMounts``` is seemed to be correct```{}```(in README), but ```[]``` in values.

fpm parameters are likely to make the same mistake.